### PR TITLE
Fix `sticky-sidebar` on repo root

### LINKED ISSUE #5008: `sticky-sidebar` no longer works on repository root
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.

### Include in this issue:

- [ ] Screenshots/video/gif demonstrating the bug, if it’s visual
- [ ] Console errors, if any

### Description

Open a repository root and resize the browser window, notice the following error:

```
Uncaught TypeError: Cannot read properties of undefined (reading 'classList')
    at refined-github.js:6899
    at refined-github.js:5089
```

https://github.com/refined-github/refined-github/blob/c7bd75628a79443fb2c501a8a0ddd2d7110c3587/source/features/sticky-sidebar.tsx#L11

The selector `.repository-content .flex-column > .flex-shrink-0 > [data-pjax]` does not seem to exist anymore.

### Example URL

https://github.com/octocat/octocat.github.io (it has a slim sidebar so you can test it even on small screens)

### Browser(s) used

Chrome

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fixes #5008 

## Test URLs
 * this page (PR conversation sidebar)
 * [issue page](https://togithub.com/refined-github/refined-github/issues/4941)
 * [repo root](https://github.com/refined-github/refined-github)

## Screenshot
image

### CODE REVIEW

File: source/features/sticky-sidebar.tsx [hunk 1]
```diff
@@ -7,8 +7,8 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 
 const deinit: VoidFunction[] = [];
-// Both selectors are present on conversation pages so we need to discriminate
-const sidebarSelector = pageDetect.isRepoRoot() ? '.repository-content .flex-column > .flex-shrink-0 > [data-pjax]' : '#partial-discussion-sidebar';
+
+const sidebarSelector = '.Layout-sidebar > div';
```
Comment (cheap-glitch) [inline source/features/sticky-sidebar.tsx:11]: Both pages (repo root & conversations) have a similar layout now so we can just use a single selector \o/

Comment (fregante) [reply source/features/sticky-sidebar.tsx:11]: Is there a more specific selector?

Comment (cheap-glitch) [reply source/features/sticky-sidebar.tsx:11]: How about
```css
.Layout-sidebar > :is(.Border-grid, #partial-discussion-sidebar)
```
?

(First one in parentheses is for repo root, second one for conversation pages.)

Comment (fregante) [reply source/features/sticky-sidebar.tsx:11]: It looks a little better (especially with comments in the code)

Comment (fregante) [reply source/features/sticky-sidebar.tsx:11]: It'd be good to merge this PR by v21.11.1

**Comment by cheap-glitch** [comment]
Any one else finds this blank space weird? The sidebar should probably be closer to the top of the window. It makes sense to have it on conversation pages, because of the sticky header, but not on the repo root imo.

image

**Pull request merged**